pkgs/top-level/all-packages.nix

```diff
@@ -522,6 +522,8 @@ with pkgs;
 
   crow-translate = libsForQt5.callPackage ../applications/misc/crow-translate { };
 
+  cy = callPackage ../tools/misc/cy { };
```

Comment:
dont add this. how does this even work since your PR added it to by-name?

Reply:
that was where I originally had it, forgot to change it in all-packages once i moved it to by-name, should be fixed now